### DISCUSSION TITLE
How do i resolve this pod install error on flutter?

### DISCUSSION THREAD

**Discussion by FurkannCetin** [drknzz/GitHub-Achievements#368] [Q&A]
I can't load all the requested pods apparently there is a problem with these two project dependencies ... "pod upgrade" terminal command results are:

Updating local specs repositories
Analyzing dependencies
firebase_auth: Using Firebase SDK version '10.3.0' defined in 'firebase_core'
firebase_core: Using Firebase SDK version '10.3.0' defined in 'firebase_core'
firebase_database: Using Firebase SDK version '10.3.0' defined in 'firebase_core'
[!] CocoaPods could not find compatible versions for pod "Firebase/Database":
  In Podfile:
    firebase_database (from `.symlinks/plugins/firebase_database/ios`) was resolved to 10.0.7, which depends on
      Firebase/Database (= 10.3.0)

    flutter_geofire (from `.symlinks/plugins/flutter_geofire/ios`) was resolved to 0.0.1, which depends on
      GeoFire (~> 4.0) was resolved to 4.1.0, which depends on
        Firebase/Database (~> 6.0)

**Comment by uvindu94** [comment]
<h1>Pod Upgrade Issue: Firebase/Database</h1>
    <p>When running the <code>pod upgrade</code> command, you may encounter an issue with the following dependencies:</p>
    <ul>
        <li><code>firebase_auth</code>: Using Firebase SDK version '10.3.0' defined in 'firebase_core'</li>
        <li><code>firebase_core</code>: Using Firebase SDK version '10.3.0' defined in 'firebase_core'</li>
        <li><code>firebase_database</code>: Using Firebase SDK version '10.3.0' defined in 'firebase_core'</li>
    </ul>
    <p>The error message may look like this:</p>
    <pre>
[!] CocoaPods could not find compatible versions for pod "Firebase/Database":
In Podfile:
firebase_database (from .symlinks/plugins/firebase_database/ios) was resolved to 10.0.7, which depends on
Firebase/Database (= 10.3.0)
    </pre>
    <h2>Solution</h2>
    <p>To resolve this issue, you can try the following steps:</p>
    <ol>
        <li>Ensure that your <code>Podfile</code> is pointing to compatible versions of Firebase pods. Update your <code>Podfile</code> as needed.</li>
        <li>Run <code>pod repo update</code> to update your local specs repositories.</li>
        <li>Then, run <code>pod install</code> or <code>pod update</code> to reinstall the pods.</li>
        <li>If the issue persists, consider specifying an exact compatible version of <code>Firebase/Database</code> in your <code>Podfile</code>.</li>
    </ol>
    <p>If you continue to experience problems, consult the <a href="https://firebase.google.com/docs/ios/setup" target="_blank">Firebase iOS setup documentation</a> or reach out to the community for support.</p>
